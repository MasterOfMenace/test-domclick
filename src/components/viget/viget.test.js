import React from 'react';
import renderer from 'react-test-renderer';
import Viget from './viget.jsx';

const mock = {
  max: 10,
  min: 0,
  start: 2
};

it(`Viget component render correctly`, () => {
  const tree = renderer
    .create(
        <Viget
          start={mock.start}
          max={mock.max}
          min={mock.min}
          onChange={()=>{}}
        />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
