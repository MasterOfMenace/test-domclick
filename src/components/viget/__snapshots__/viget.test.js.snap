// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Viget component render correctly 1`] = `
<div>
  <button
    onClick={[Function]}
  >
    -
  </button>
  <input
    max={10}
    min={0}
    onBlur={[Function]}
    onChange={[Function]}
    onFocus={[Function]}
    type="number"
    value={2}
  />
  <button
    onClick={[Function]}
  >
    +
  </button>
</div>
`;
